pulsar-broker/src/main/java/org/apache/pulsar/PulsarBrokerStarter.java

```diff
@@ -44,8 +55,79 @@ private static ServiceConfiguration loadConfig(String configFile) throws Excepti
         return config;
     }
 
+    private static final Options OPTS = new Options();
```

Comment:
We use JCommander for parsing CLI options in the other tools, eg: see https://github.com/apache/incubator-pulsar/blob/45ea0bc45e67751711708ecd7ba8f446ff34c67a/pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandaloneStarter.java#L52

Reply:
thanks, will change it.